ament_package/__init__.py

```diff
@@ -113,20 +113,20 @@ def parse_package_string(data, *, filename=None):
     nodes = _get_nodes(root, 'package')
     if len(nodes) != 1:
         raise InvalidPackage("The manifest must contain a single 'package' "
-                             "root tag")
+                             'root tag')
```

Comment:
Nit: wrap after open parenthesis and keep the string on a single line

Reply:
Well, this is already existing code which wraps like this. So it could just stay this way. Or you could just do the edit in the PR... :wink:

Anyway fixed in 4fbc6f4d1c7bf44c26f52f9c43d8716f2a7eecf4.

Reply:
agree it's just that it stroke me that half the changes in this PR were on the same lines as "not preferred" wrapping. But it can be done in a follow-up if you don't think it belongs in this PR